### PR TITLE
Finish all basic members and people imports

### PR DESCRIPTION
**Pull request by facundobatista**
All changes for final import with all data needed; after running the imports from CSVs in this branch, everything from the spreadsheet should be in DB.

IMPORTANT: changes in the model! 

- legal_id and registration_date are now nullable! this is because we need to create the member for a person and start storing some data there (like category, or if the student certificate is presented) even if the membership is not yet "accepted". 

- membership in Person now can NOT be null, as there will always been a Persona and a Member as a unique pair, one can not be alive without the other

- other minor stuff: added the plural for Category (for the admin), and some aesthetic changes
    
The rest of the branch includes:

- Improved the import scripts:

    - Started accessing the row with column names (in members only, people was already there)

    - No more "get or create" for person, patrons and members, as that hid some issues

    - Added proper fees and descriptions to categories

    - Split the address single string in its components (street, city, etc)

    - Started supporting the weird format dates from real spreadsheet

    - Read all the flags from "people" CSV

- Added Category to the admin interface (plus some aesthetic changes there)

### CODE REVIEW

File: website/members/management/commands/import_members.py [hunk 1]
```diff
@@ -1,13 +1,76 @@
...
+
+        25-abr.-2018
+        1-jul.-2016
+    """
+    day, monthname, year = date_string.split("-")
+    return datetime.date(year=int(year), month=MONTHS[monthname], day=int(day))
+
+
+def split_address(complete):
+    """Split one line of address in its components."""
```
Comment (cmdelatorre) [inline website/members/management/commands/import_members.py]: Estaría bueno describir el formato esperado de este string `complete`. ¿de dónde sale? ¿dónde se define? ¿es algo escrito a mano (error prone) o lo genera automáticamente otra herramienta?
Entre otras cosas, puede ser útil en caso de que haga falta debuggear.

File: website/members/management/commands/import_members.py [hunk 2]
```diff
@@ -1,13 +1,76 @@
...
+
+    m = re.match("(.*) \((.*)\)", city_pc)
+    if m:
+        city, postcode = m.groups()
+    else:
+        city, postcode = ("", "")
+
+    if "" in (complete, streetadd, city, prov, country):
+        print("======== address", (complete, streetadd, city, postcode, prov, country))
+    return streetadd, city, postcode, prov, country
```
Comment (cmdelatorre) [inline website/members/management/commands/import_members.py]: Devolver tuplas suele ser poco robusto. Un dict es mucho más explícito.

File: website/members/management/commands/import_people.py [hunk 3]
```diff
@@ -1,10 +1,60 @@
...
+    language-dependant ordering, e.g.:
+
+        25-abr.-2018
+        1-jul.-2016
+    """
+    day, monthname, year = date_string.split("-")
+    return datetime.date(year=int(year), month=MONTHS[monthname], day=int(day))
+
+
+def split_address(complete):
```
Comment (cmdelatorre) [inline website/members/management/commands/import_people.py]: huf, todo este copy paste acá... ¿estos scripts son _one-timers_, cierto? Es para popular la DB por primera vez?

File: website/members/management/commands/import_members.py (hunk 1 above)
Comment (facundobatista) [reply website/members/management/commands/import_members.py]: Es algo que se va a hacer y usar solamente en la carga inicial, porque en la planilla web tenemos todo junto. No vale la pena mayor laburo ahí.

File: website/members/management/commands/import_members.py (hunk 2 above)
Comment (facundobatista) [reply website/members/management/commands/import_members.py]: Sí, pero de nuevo, esto es usar y descartar

File: website/members/management/commands/import_people.py (hunk 3 above)
Comment (facundobatista) [reply website/members/management/commands/import_people.py]: Xacto

**Pull request merged**